programs/cryptid_signer/src/instruction/001_propose_transaction.rs

```diff
@@ -1,13 +1,13 @@
-use std::mem::{size_of, swap};
-use std::num::NonZeroU64;
+use std::ops::DerefMut;
```

Comment:
I'd like to see some more comments in this file - especially in the longer functions.